layouts/partials/scroll-to-top.html

```diff
@@ -0,0 +1,6 @@
+<div class="scroll-container">
+  <button id="scrollToTopBtn" class="scroll-btn btn" aria-label="Scroll to top" title="Scroll to top">
+    <i class="fas fa-chevron-up scroll-icon"></i>
+    <span class="scroll-label">Back to Top</span>
```

Comment:
this is never visible for me

Reply:
It comes up after 3 to 4 scrolls which is mostly for long content pages did you try such page, maybe the registry page? @svrnm

Reply:
what's the purpose of showing the arrow earlier than the text?

Reply:
For the sake of i18n, I'd drop the English label. I think that using an icon is clear enough. WDYT @svrnm?

Reply:
As pointed out by @theletterf i ought to only display the text after 3-4 page scrolls so I decided to only display the arrow until the page scroll triggers displaying the text? Or how do you suggest I go about it @svrnm

Reply:
having that label is an accessibility feature for me, not everyone might immediately understand the meaning of this button/we can not assume that since we understand it everyone does. And again, that's also why I dislike the 3-4 page scrolls until it shows up. 

Although I have to admit I didn't think about the localized pages.

Since we have conflicting opinions between maintainers here, I suggest we do not over-engineer the problem for now and block @bintus-ux any longer, so let's do the following:

* Remove the "Back to top" label for now 
* Raise a new issue that we want to have it added to the scroll to top functionality. 

For me this would be good enough, wdyt? @chalin @theletterf

Reply:
Yes, good plan! 🙌🏻

Reply:
Btw, the need-of-rebase issue and one suggestion I made in https://github.com/open-telemetry/opentelemetry.io/pull/5620#pullrequestreview-2482825231 are still pending.

Reply:
Yea that sounds good, fixes will be done today, thanks again!